### ISSUE TITLE
JVM crashes when using docker maven plugin

### ISSUE THREAD

**Issue by kosio197** [fabric8io/docker-maven-plugin#1126]
### Description
When I'm using maven docker plugin I have the following fail:
[INFO] --- docker-maven-plugin:0.20.1:build (build) @ demo ---
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x0000000000000000, pid=13869, tid=0x00007f72deef4700
#
# JRE version: Java(TM) SE Runtime Environment (8.0_192-b12) (build 1.8.0_192-b12)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.192-b12 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# C  0x0000000000000000
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /home/kosio/eclipse-workspace/spring-boot-postgresql-docker-compose/hs_err_pid13869.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
Aborted

During trying to simplify the use case I tried with this project and the result is the same: "https://github.com/jirkapinkas/spring-boot-postgresql-docker-compose"

I'm able to start postgres container from the command line using:
docker run -d -p6432:5432 --name test_doc -e POSTGRES_PASSWORD=test_pass -e POSTGRES_USER=test_user -e POSTGRES_DB=test_db postgres 

### Info

* d-m-p version :0.20.1
* Maven version (`mvn -v`) :
```
Apache Maven 3.6.0 (97c98ec64a1fdfee7767ce5ffb20918da4f719f3; 2018-10-24T21:41:47+03:00)
Maven home: /home/kosio/work/apache-maven-3.6.0
Java version: 1.8.0_192, vendor: Oracle Corporation, runtime: /usr/lib/jvm/jdk1.8.0_192/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "4.19.3", arch: "amd64", family: "unix"

```
* Docker version : Docker version 17.05.0-ce, build 89658be
* If it's a bug, how to reproduce : I'm hitting this in every project which uses maven docker plugin, including the one in the description
* If it's a feature request, what is your use case :
* Sample project : *[GitHub Clone URL]* https://github.com/jirkapinkas/spring-boot-postgresql-docker-compose

The operation system is Debian 9
[hs_err_pid13869.log](https://github.com/fabric8io/docker-maven-plugin/files/2609319/hs_err_pid13869.log)

**Comment by rhuss** [comment]
Thanks for the bug report, but I really have no idea what causes this. The sample project you posted works fine for me.

Also, at d-m-p is a pure  Java-based implementation, I'm pretty sure that d-m-p is not the root cause but triggers a bug elsewhere (e.g. the JVM). Unfortunately, a full crash dump analysis of the memory dump is out of scope of what I'm able to support.
